apps/web/src/pages/api/traces.ts

```diff
@@ -2,23 +2,26 @@ import type { NextApiRequest, NextApiResponse } from "next";
 import { NextApiHandler } from "next";
 import { NextRequest, NextResponse } from "next/server";
 import { api } from "src/utils/api";
+import { PrismaClient } from "@prisma/client";
 // change to get from prisma
 
 export default function handler(req: NextApiRequest, res: NextApiResponse) {
-  // if (req.method === "POST") {
-  //   chains.push(req.body.data.chain);
-  //   res.status(200).send(null);
-  //   return;
-  // }
-
-  const query = api.chain.getTraces.getQueryKey();
-  //@ts-ignore
-  const data = query.data ?? [];
-  const chains: any[] = [];
-  for (const d of data) {
-    chains.push(JSON.parse(d.trace));
+  if (req.method === "POST") {
```

Comment:
hey! just wondering why this handler if we also use trpc? is this just so we don't need to worry about setting up openapi integration?

Reply:
i'll try rn to see if it's not too complex to set up openapi for it though

Reply:
I've done this with openapi and can help here to save you time. Don't worry about it for now. This is good!